wand/frontend/wand_server.py

```diff
@@ -250,8 +252,16 @@ async def lock_task(self, laser):
                     v_pzt = iface.get_pzt_voltage()
                     v_pzt -= V_error
 
-                    if v_pzt > 100 or v_pzt < 25:
-                        logger.warning("'{}'' lock railed".format(laser))
+                    if v_pzt > v_pzt_max or v_pzt < v_pzt_min:
+                        logger.warning(
+                            "'{}'' lock railed, piezo voltage: {:.2f}V outside range {} - {}V"
```

Comment:
stray `'`?

Reply:
Looks the same as before, though I think there was previously a stray `'`.
Can fix it though

Reply:
ack. Do you mind fixing. Otherwise looks good to merge.